TTHAnalysis/python/plotter/ttH-multilepton/mca-2lss-mc.txt

```diff
@@ -1,18 +1,18 @@
-ttH+    : TTHnobb_mWCutfix_ch0 : 0.5071*0.418 : LepGood1_isMatchRightCharge && LepGood2_isMatchRightCharge; FillColor=ROOT.kOrange+10
```

Comment:
Are you sure these commented lines does not mess up things for other people? Perhaps we should have a different mca, for the moment, or use the one used by everyone?

@sscruz